mountpoint-s3-client/src/lib.rs

```diff
@@ -51,7 +51,7 @@ pub mod failure_client;
 pub mod imds_crt_client;
 #[doc(hidden)]
 pub mod mock_client;
-mod object_client;
+pub mod object_client;
```

Comment:
Revert and instead re-export the required type(s) explicitly under the `types` module (see from line 68).